cwltool/stdfsaccess.py

```diff
@@ -44,3 +49,9 @@ def join(self, path, *paths):  # type: (Text, *Text) -> Text
 
     def realpath(self, path):  # type: (Text) -> Text
         return os.path.realpath(path)
+
+    # On windows os.path.realpath appends unecessary Drive, here we would avoid that
+    def docker_compatible_realpath(self, path):  # type: (Text) -> Text
+        if onWindows():
+            return path
```

Comment:
Why is `realpath` being called in the first place? Seems dangerous to do nothing if `onWindows()`..